pod/perldeprecation.pod

```diff
@@ -60,6 +60,18 @@ in deprecation warnings.
 
 Category: "deprecated::unicode_property_name"
 
+=head2 Perl 5.46
+
+=head3 C<use VERSION> while another C<use VERSION> is in scope
+
+A C<use VERSION> declaration has many implicit effects on the surrounding
+scope, such as L<strict> and L<feature> flags, or importing L<builtin>
+functions. Once you have a C<use VERSION> statement in scope, any other
+C<use VERSION> statement is now deprecated since Perl 5.39.8, due to the
+increasing complexity of swapping from one prevalent version to another.
```

Comment:
Typo: "prevalent" should be "prevailing". Also in `perldiag.pod`.

Reply:
Oops; yes I think I was even the one who came up with the word "prevailing" in the first place.